src/main/java/org/terasology/blockNetwork/ImmutableBlockLocation.java

```diff
@@ -36,11 +37,11 @@
      * z-coordinate of the location.
      */
     public final int z;
-    
-    public ImmutableBlockLocation(Vector3i location) {
-        this(location.x, location.y, location.z);
+
+    public ImmutableBlockLocation(Vector3ic location) {
```

Comment:
Can this whole class just be removed in favor of `Vector3ic`?

Reply:
I can but the module doesn't really work and the way it works is very limited. I rather just make the changes to drop the dependencies to the deprecated methods.